CHANGELOG.md

```diff
@@ -57,6 +57,7 @@ https://github.com/certbot/certbot/milestone/62?closed=1
 * The grammar used by Augeas parser in Apache plugin was updated to fix various parsing errors.
 * The CloudXNS, DNSimple, DNS Made Easy, Gehirn, Linode, LuaDNS, NS1, OVH, and
   Sakura Cloud DNS plugins are now compatible with Lexicon 3.0+.
+* Only depend on imgconverter extension for Sphinx >= 1.6
```

Comment:
It looks like we had a bad merge here. This should be added to the 0.29.0 section above.